include/NovelRT/Utilities/Misc.h

```diff
@@ -4,8 +4,14 @@
 #ifndef NOVELRT_UTILITIES_MISC_H
 #define NOVELRT_UTILITIES_MISC_H
 
-#ifndef NOVELRT_H
-#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#include <filesystem>
+#include <gsl/span>
+#include <type_traits>
+
+#if defined(NDEBUG)
+#define unused(x) (void)(x)
+#else
+#define unused(x) (void)(0)
```

Comment:
Is this being used somewhere?

Reply:
I remember a reason for this but don't remember that reason.

With that said, we could probably use this and make this `static_cast<void>` to make it clearer, if we do want it.